src/components/layout/playground/requests/requestCard.tsx

```diff
@@ -174,7 +177,11 @@ const PlaygroundRequestCard: React.FC<
               canEdit ? 'w-1/2' : ''
             }`}
           >
-            Read more / apply
+            Read more{' '}
+            {session?.user?.role !== 'Admin' &&
+            requester?.id !== session?.user?.id
+              ? '/ apply'
+              : null}
```

Comment:
```suggestion
                        {`Read more${
              session?.user?.role !== 'Admin' &&
              requester?.id !== session?.user?.id
                ? ' / apply'
                : ''
            }`}
```
Somehow the other solution is adding comments under the text